tests/tests/control_wrtGauss_netcdf_parallel

```diff
@@ -34,7 +34,7 @@ export WRITE_DOPOST=.true.
 export IAER=5111
 export IOVR=3
 export OUTPUT_FILE="'netcdf_parallel' 'netcdf_parallel'"
-export IDEFLATE=1
+export ZSTANDARD_LEVEL=5
 export QUANTIZE_NSD=14
```

Comment:
@DusanJovic-NOAA Can you confirm the value of QUANTIZE_NSD? I think it should be 5 with "quantize_bitround'.

Reply:
With "quantize_bitround" QUANTIZE_NSD is a number of significant binary digits, so value 14 corresponds to 14 significant bits, which is what we used previously with built-in quantization (nbits was the name of the parameter if I remember correctly).

Reply:
Thanks for confirming.